gateway/1.6/modules/ROOT/pages/policies-included-traffic-management-route.adoc

```diff
@@ -0,0 +1,175 @@
+= Traffic Management for Multiple Upstream Services
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:keywords: route, traffic management, upstream service
+
+[%autowidth.spread,cols="a,a"]
+|===
+>s| Policy Name | Traffic Management for Multiple Upstream Services
+>s|Summary      | Manages API instance traffic to multiple upstream services from a single consumer endpoint
+>s| First Flex Gateway version available | v1.0.0
+.4+>.^s| Returned Status Codes | No return codes exist for this policy.
+[%autowidth.spread,cols="a,a"]
+|===
+
+[NOTE]
+====
+The following information applies only to Flex Gateway running in Local Mode. To configure multiple upstream services for Flex Gateway running in Connected Mode, see xref:api-manager::create-instance-task-flex.adoc#traffic-management[Multiple Upstream Services for Flex Gateway Running in Connected Mode].
+====
+
+== Summary
+
+Flex Gateway manages request traffic by using different routes that can each direct traffic to multiple upstream services. Flex Gateway directs traffic to the routes by using the route order and the individual route's rules. Additionally, you can add a weighted percentage to each upstream service within a route to manage the percentage of requests sent to the upstream service. For information about adding weight percentages, refer to xref:policies-included-traffic-management.adoc[].
+
+To allow routing requests to a `Service` when an `ApiInstance` does not include `spec.services`, you must define a `PolicyBinding` configuration with the Traffic Management for Multiple Upstream Services (`route`) policy. It is not necessary to configure a `route` policy if a custom policy is managing requests.
+
+== Configuring Policy Parameters
+
+Refer to the following policy definition and table of parameters:
+
+----
+policyRef:
+  name: route
+config:
+  destinationPath: <string> // OPTIONAL, default: "/"
+  destinationRef:
+      name: <string> // REQUIRED
+      kind: <string> // OPTIONAL, default: "Service"
+      namespace: <string> // OPTIONAL, default: ""
+rules:
+  - path: <URL-regex> // OPTIONAL, Example: (.*)
+    methods: <regex-string> // OPTIONAL, Example: GET|POST
+    headers:
+      <header-name>: <regex-string> // OPTIONAL
+    host: <regex-string> // OPTIONAL
+----
+
+[%header%autowidth.spread,cols="a,a,a,a"]
+|===
+| Parameter | Required or Optional | Default Value | Description
+
+| `destinationPath`
+| Optional
+| "/"
+| Path of the upstream service
+
+| `destinationRef.name`
+| Required
+| N/A
+| Name of the upstream service as defined in the `Service` configuration resource
+
+| `destinationRef.kind`
+| Optional
+| "Service"
+|
+
+| `destinationRef.namespace`
+| Optional
+| ""
+|
+
+| `rules`
+| Optional
+| Empty
+| Array of rulesets for this route. +
+Each route can support multiple rulesets.
+
+| `rules.path`
+| Optional
+| Empty
+| Request path that the route can service. +
```

Comment:
```suggestion
| Request path that the route can service. 
+
```

Reply:
(I don't know if this even works in tables, but consider putting the plus on a line by itself for more vertical space.